test/generated_inputs/BUILD.bazel

```diff
@@ -0,0 +1,76 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary", "rust_library", "rust_test")
+
+genrule(
```

Comment:
Can you use [@bazel_skylib//rules:write_file.bzl](https://github.com/bazelbuild/bazel-skylib/blob/main/docs/write_file_doc.md) instead of `genrule` here?

Reply:
Done.